src/instructlab/lab.py

```diff
@@ -126,16 +126,29 @@ def cli(ctx, config):
     "Please do not use this option if you are planning to contribute back "
     "using the same taxonomy repository. ",
 )
+@click.option(
+    "workspace_path",
+    "--workspace-path",
+    type=click.Path(),
+    envvar="ILAB_WORKSPACE_PATH",
+    help="Path to the workspace directory. Creates it if it doesn't exist.",
+)
 def init(
     interactive,
     model_path,
     taxonomy_path,
     taxonomy_base,
     repository,
     min_taxonomy,
+    workspace_path,
 ):
     """Initializes environment for InstructLab"""
 
+    if workspace_path:
+        if not exists(workspace_path):
+            os.mkdir(workspace_path)
+        os.chdir(workspace_path)
```

Comment:
I had another thought about the UX for a workspaces concept. I think the use of `os.chdir()` results in some surprising behavior.  Take this for example:

```
ilab init --taxonomy-path relative/path/to/taxonomy --workspace-path path/to/workspace
```

Any time I specify a relative path, I would expect it to be interpreted relative to the working directory where I'm running the command.

For my example, I would expect `ilab` to take my relative path and write an absolute path into the config file in the workspace that points it to the right place. I think other surprises could happen when the tool is switching its working directory out by surprise.

In other words, while it's more code, I think less surprising behavior would be to not change working directories, but instead, do more careful handling of paths throughout using that workspace path value.